### PR TITLE
Text transcription and audio integration classes and methods added

### PR DESCRIPTION
**Pull request by n400peanuts**

### CODE REVIEW

File: tests/test_audio_transcription.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    audio_transcriber = AudioTranscriber(language='dutch')
+    filepath = os.path.join('tests', 'audio_files', 'test_dutch_5_seconds.wav')
+
+    with open(os.path.join(
+            'tests', 'reference_files', 'transcription_dutch_1_second.json'), 'r') as file:
+        ref_transcription = json.loads(file.read())
+
+    def test_apply(self):
+        transcription = self.audio_transcriber.apply(self.filepath)
+        assert are_equal(transcription['transcription'], self.ref_transcription['transcription'])
```
[the remark this answers is not in the document]
Comment (n400peanuts) [reply tests/test_audio_transcription.py:17]: makes sense! Haven't thought about it

File: tests/test_audio_text_integration.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    def test_apply(self):
+
+        out  = self.audio_text_integrator.apply(self.audio_filepath)
+
+        assert are_equal(out, self.text_audio_transcription) # test the output against the reference
+
+
+
+def are_equal(reference_output, predicted_output):
```
[the remark this answers is not in the document]
Comment (n400peanuts) [reply tests/test_audio_text_integration.py:25]: I am not sure I follow here. 
The function are_equal() tests whether the two dictionaries are the same in each value. I can move it to test_apply -- though this is already called there, so I am confused here.

[the remark this answers is not in the document]
Comment (n400peanuts) [reply tests/test_audio_text_integration.py:25]: also, while test_apply() is rather a generic type of call which means that we're applying "a" test, in this case the test is about whether the two dictionaries are the same (i.e., are_equal()). In another situation you may want to test something else?

**Review by maltelueken — APPROVED** [review]
Looks good now!

**Pull request merged**